nifi-commons/nifi-utils/src/main/java/org/apache/nifi/processor/util/StandardValidators.java

```diff
@@ -390,6 +391,19 @@ public ValidationResult validate(final String subject, final String input, final
         }
     };
 
+    public static final Validator URI_LIST_VALIDATOR = (subject, input, context) -> {
+
+        if (context.isExpressionLanguageSupported(subject) && context.isExpressionLanguagePresent(input)) {
+            return new ValidationResult.Builder().subject(subject).input(input).explanation("Expression Language Present").valid(true).build();
+        }
+        Optional<ValidationResult> invalidUri = Arrays.stream(input.split(","))
```

Comment:
I believe the way this stream is set up that an input of empty string would be considered valid.

Reply:
Good catch, will add a check for null/empty before I start the stream